src/adapters/glexchange-polygon-1/index.ts

```diff
@@ -0,0 +1,92 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+import BigNumber from "bignumber.js";
+import moment from "moment";
+import { EventData } from "web3-eth-contract";
+import path from "path";
+import priceSdk from "../../sdk/price";
+import Matic from "../../sdk/matic";
+import symbolSdk from "../../sdk/symbol";
+import { ISaleEntity, ISymbolAPIResponse } from "../../sdk/Interfaces";
+
+class GlExchange {
+    name: string;
+    token: string;
+    protocol: string;
+    block: number;
+    contract: string;
+    events: string[];
+    pathToAbi: string;
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "glexchange-polygon-1";
+        this.protocol = "polygon";
```

Comment:
Update protocol to `matic`. We don't support `polygon` yet.

Reply:
Changed protocol to matic . but in your sdk you have mentioned as polygon  here https://github.com/dappradar/nft-sales-adapters/blob/main/src/sdk/constants.ts#L4